src/Nelmio/Alice/Instances/Processor/Methods/ArrayValue.php

```diff
@@ -44,6 +44,7 @@ public function canProcess(ProcessableInterface $processable)
      */
     public function process(ProcessableInterface $processable, array $variables)
     {
+        /** @var array $values */
         $values = $processable->getValue();
```

Comment:
should be a comment not a PHPDoc block (`/* @var ... */` instead of `/** @var ... */`)